core/domain/classifier_domain_test.py

```diff
@@ -181,6 +182,71 @@ def test_validation_with_invalid_training_data(self) -> None:
             utils.ValidationError, 'Expected training_data to be a list'):
             training_job.validate()
 
+    def test_classifier_data_filename(self) -> None:
+        expected_filename = 'exp_id1.SOME_RANDOM_STRING-classifier-data.pb.xz'
+        observed_filename = self._get_training_job_from_dict(
+            self.training_job_dict).classifier_data_filename
+        self.assertEqual(expected_filename, observed_filename)
+
+    def test_status_update_with_correct_status(self) -> None:
+        training_job = self._get_training_job_from_dict(self.training_job_dict)
+        training_job.update_status('PENDING')
+
+    def test_status_update_with_invalid_status(self) -> None:
+        training_job = self._get_training_job_from_dict(self.training_job_dict)
+        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
+            Exception,
+            'The status change %s to %s is not valid.' %
+            ('NEW', 'invalid_status')):
+            training_job.update_status('invalid_status')
+
+    def test_attributes(self) -> None:
+        training_job = self._get_training_job_from_dict(self.training_job_dict)
+        self.assertEqual(
+            self.training_job_dict['job_id'],
+            training_job.job_id
+            )
```

Comment:
Deindent by 4 to match opening paren. Ditto below.

Reply:
Done 😃